Plugins/InstallSafeDITool/InstallCLIPluginCommand.swift

```diff
@@ -75,7 +78,7 @@ struct InstallSafeDITool: CommandPlugin {
             let downloadedFileAttributes = try FileManager.default.attributesOfItem(atPath: downloadedURL.path())
             guard let currentPermissions = downloadedFileAttributes[.posixPermissions] as? NSNumber,
                   // Add executable attributes to the downloaded file.
-                  chmod(downloadedURL.path(), mode_t(currentPermissions.uint16Value | S_IXUSR | S_IXGRP | S_IXOTH)) == 0
+                  chmod(downloadedURL.path(), mode_t(currentPermissions.uint32Value) | S_IXUSR | S_IXGRP | S_IXOTH) == 0
```

Comment:
On Linux `mode_t`  is a `UInt32`. On Apple's platforms it is a `UInt16`. So I need to wrap the `mode_t` cast on the `currentPermissions` value alone, and I figured I'd downscale the `UInt32` value rather upscaling the `UInt16` value. Doesn't really make a difference in the end though.